WooCommerce/UITestsFoundation/Screens/TabNavComponent.swift

```diff
@@ -48,7 +48,7 @@ public final class TabNavComponent: ScreenObject {
     @discardableResult
     public func gotoOrdersScreen() throws -> OrdersScreen {
         // Avoid transitioning if it is already on screen
-        if !OrdersScreen.isVisible {
+        if !OrdersScreen.isLoaded() {
```

Comment:
Because of reasons mentioned in nearby comments, this can be reverted:

```suggestion
        if !OrdersScreen.isVisible {
```

`isVisible` actually calls the `isLoaded` from `ScreenObject`